Fix EmailStr for Cython (#1126)
samuelcolvin/pydantic: changes/1126-koxudaxi.md

> Fix EmailStr for Cython with when used in Linux.

Reviewer: ```suggestion
Fix `EmailStr` for Cython when used in Linux to accept instances of `EmailStr`
```

> Fix `EmailStr` for Cython when used in Linux to accept instances of `EmailStr`

Reviewer: ```suggestion
Fix `EmailStr` and `NameEmail` to accept instances of themselves in cython.
```
